peer.go

```diff
@@ -742,7 +742,36 @@ out:
 			}
 
 		case *lnwire.Error:
-			p.server.fundingMgr.processFundingError(msg, p.addr)
+			// Check if the error specifies an all-zero channel ID.
+			isZeroID := true
+			for _, b := range msg.ChanID {
```

Comment:
This can be simplified by having something like: 
```go
type ConnectionWideError ChannelID
```
in `lnwire`, 

then this would just be: 
```go
isZeroID := msg.ChanID == ConnectionWideError
```

Reply:
Added variable `var ConnectionWideID = ChannelID{}` as I imagine this all-zero ID also could be used for other messages than errors.